src/js/samples/components/Tag/WorkflowTagNone.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+import { Icon } from "../../../base";
+import { BaseWorkflowTag } from "./BaseWorkflowTag";
+import { WorkflowLabelIcon } from "./WorkflowLabelIcon";
+
+const StyledSampleItemWorkflowTagNone = styled(BaseWorkflowTag)`
+    background-color: ${props => props.theme.color.purpleLightest};
+    border: 1px solid ${props => props.theme.color.purple};
+    color: ${props => props.theme.color.purpleDarkest};
+`;
+
+/**
+ * Display a "no analyses" tag.
```

Comment:
Where? When would you want to show this?

```suggestion
 * A  "no analyses" tag.
```